src/BuiltInTools/dotnet-watch/HotReload/RestartPrompt.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Microsoft.Build.Tasks;
-
 namespace Microsoft.DotNet.Watch
```

Comment:
Should this also include making the `namespace` declarations inline?

Reply:
I only made file-scoped namespaces for the `dotnet` and `Cli.Utils` projects. This file is in neither of those.